content/issues/managing-your-tasks-with-tasklists/creating-a-tasklist.md

```diff
@@ -46,8 +46,8 @@ You can also use the **Add tasklist** button while editing your issue to insert
 Once you have started editing your tasklist Markdown, you can add new tasks by preceding each new task with `- [ ] ` and then adding either:
 
 - A draft task. Draft tasks are text that can later be converted into issues.
-- The full link to an issue or pull request or, if the issue or pull request is in the same repository as your tasklist, you can use the #ISSUE-NUMBER shorthand syntax. For example, `https://github.com/octo-org/octo-repo/issues/45` or `#45`.
-- If an issue or pull request is in the same repository
+- The full link to an issue or pull request.  For example, `https://github.com/octo-org/octo-repo/issues/45`.
```

Comment:
Merged the 2nd and 3rd bullet points as otherwise the third reads weirdly when in conjunction with the "Once you have started..." sentence
```suggestion
- The full link to an issue or pull request.  For example, `https://github.com/octo-org/octo-repo/issues/45`. If an issue or pull request is in the same repository as your tasklist, you can use the #ISSUE-NUMBER shorthand syntax. For example, `#45`.
```